geos/cmake.win.patch

```diff
@@ -1,11 +1,11 @@
---- capi\CMakeLists.txt.orig	Wed Dec 10 18:54:08 2014
-+++ capi\CMakeLists.txt	Wed Dec 10 18:53:35 2014
+--- capi\CMakeLists.txt.orig	Sun Aug 25 15:10:32 2013
++++ capi\CMakeLists.txt	Mon Jan 19 15:10:24 2015
 @@ -25,7 +25,7 @@
-   # if building OS X framework, CAPI built into C++ library
-   add_library(geos_c SHARED ${geos_c_SOURCES})
-
+   # if building OS X framework, CAPI built into C++ library 
+   add_library(geos_c SHARED ${geos_c_SOURCES}) 
+ 
```

Comment:
I think your installation of Git fudges with the config. I installed git on a clean system, and kept the default "Windows newline on clone, Linux newline on commit" option - this seems to work without a hitch on said machine.